apps/arena/lib/arena/matchmaking/game_launcher.ex

```diff
@@ -115,6 +116,8 @@ defmodule Arena.Matchmaking.GameLauncher do
         client_id: client_id,
         skin_name: "Basic",
         character_name: Enum.random(characters).name,
+        # TODO: what do we do with bot levels? Average over all real player's levels
+        character_level: 1,
```

Comment:
Nice question. Let's create an issue and link it here.

Reply:
created #1170